src/osp/Active/SysDebugRender.cpp

```diff
@@ -42,55 +58,176 @@ using namespace Magnum::Math::Literals;
 
 void SysDebugRender::add_functions(ActiveScene &rScene)
 {
-    Package& glResources = rScene.get_context_resources();
-
-    using namespace adera::shader;
-
-    glResources.add<Phong>("phong_shader",
-        Phong{Magnum::Shaders::Phong::Flag::DiffuseTexture});
-
-    glResources.add<PlumeShader>("plume_shader");
-
-    rScene.debug_render_add(rScene.get_render_order(), "debug", "", "",
-                            &SysDebugRender::draw);
+    initialize_context_resources(rScene);
+    configure_render_passes(rScene);
```

Comment:
This is no longer "add functions," just call these two manually from flight.cpp instead.

Reply:
Ah alright, do you have any ideas for conventions to make setup stuff like this more intuitive? One of my least favorite parts about APIs is when there are functions that need to be called to set up the system that you might forget. I suppose we could have `init()` functions that wrap them together into one step. It's nice for them to be called automatically though.